threshsign/src/bls/relic/BlsPublicParameters.cpp

```diff
@@ -45,6 +46,18 @@ BlsPublicParameters::BlsPublicParameters(const BlsPublicParameters &params)
   g1_copy(generator1_, params.generator1_);
   g2_copy(generator2_, const_cast<G2T &>(params.generator2_));
 }
+BlsPublicParameters& BlsPublicParameters::operator=(const BlsPublicParameters& params){
+  securityLevel_ = params.securityLevel_;
+  schemeName_    = params.schemeName_;
+  library_       = params.library_;
+  curveType_     = params.curveType_;
+  BLS::Relic::Library::Get();
+  g1_get_gen(generator1_);
+  g2_get_gen(generator2_);
+//  g1_copy(generator1_, params.generator1_);
```

Comment:
Wy are these lines commented out? Let's delete them if they aren't needed please.

Reply:
I'll do it